lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+module Menu
+
+  attr_reader :items, :food, :price
+
+  def initialize
+    @menu_items ={"Duck Rice" => 8, "Tonkotsu Ramen" => 8, "Gyoza" => 5, "Jalapeno Hamachi" => 8, "Summer Rolls" => 5, "Mango" => 3}
+  end
```

Comment:
You have an attr_reader for items, but you have called the instance variable @menu_items. Call the variable @items so that the attr_reader works. You have put an initialize method inside a module - this should be within a class, as you cannot have an instance of a module.